java-hello-world/00-create-fund-account/src/main/java/createfundaccount/ScriptCreateFundAccount.java

```diff
@@ -27,76 +28,36 @@ public static void main(String[] args) throws Exception {
             throw new IllegalArgumentException("Please set required keys in .env file.");
         }
 
-        // Create an ECSDA secp256k1 private key based on a BIP-39 seed phrase,
-        // plus the default BIP-32/BIP-44 HD Wallet derivation path used by Metamask.
-        Security.addProvider(new BouncyCastleProvider());
-
-        // BIP-39 Generate seed from seed phrase
-        byte[] seed = MnemonicUtils.generateSeed(seedPhrase, null);
-
-        // BIP-32 Derive master key
-        Bip32ECKeyPair masterKeypair = Bip32ECKeyPair.generateKeyPair(seed);
-        final int[] path = {
-            44 | Bip32ECKeyPair.HARDENED_BIT,
-            60 | Bip32ECKeyPair.HARDENED_BIT,
-            0 | Bip32ECKeyPair.HARDENED_BIT,
-            0,
-            0
-        };
-
-        // BIP-44 Derive private key of single account from BIP-32 master key + BIP-44 HD path
-        // Using m/44'/60'/0'/0/0 for compatibility with Metamask hardcoded default
-        Bip32ECKeyPair derivedKeypair = Bip32ECKeyPair.deriveKeyPair(masterKeypair, path);
-        Credentials credentials = Credentials.create(derivedKeypair);
-        String privateKeyString = credentials.getEcKeyPair().getPrivateKey().toString(16);
-        final int privateKeyStringMissingCharCount = 64 - privateKeyString.length();
-        if (privateKeyStringMissingCharCount > 0) {
-            privateKeyString = ("0").repeat(privateKeyStringMissingCharCount) + privateKeyString;
-        }
-        PrivateKey privateKey = PrivateKey.fromStringECDSA(privateKeyString);
+        // Derive private key
+        PrivateKey privateKey = Mnemonic.fromString(seedPhrase).toStandardECDSAsecp256k1PrivateKey("", 0);
```

Comment:
This is not equivalent to the code that it replaces, as it  does not use the custom derivation path `m/44'/60'/0'/0/0`

```suggestion
        PrivateKey privateKey = Mnemonic.fromString(seedPhrase).toStandardECDSAsecp256k1PrivateKeyCustomDerivationPath("", "m/44'/60'/0'/0/" + accountIndex);
```

(where `accountIndex` can be hardcoded as `0` if that's the only one we need)